src/Elcodi/Bundle/CoreBundle/ElcodiCoreBundle.php

```diff
@@ -40,4 +41,14 @@ public function getContainerExtension()
     {
         return new ElcodiCoreExtension();
     }
+
+    /**
+     * Create instance of current bundle, and return dependent bundle namespaces
+     *
+     * @return array Bundle instances
+     */
+    public static function getBundleDependencies()
+    {
+        return [];
```

Comment:
If this bundle doesn't depend on anything, is inheritance from `DependentBundleInterface` still relevant?